istack-commons/runtime/src/main/java/module-info.java

```diff
@@ -11,7 +11,7 @@
 module com.sun.istack.runtime {
     requires java.logging;
     requires transitive java.xml;
-    requires transitive java.activation;
+    requires transitive jakarta.activation;
```

Comment:
module name should not be changed

Reply:
Yes, it was intended.  Now that we've decided to use jakarta.* instead of java.*, the sooner we make the switch the better.  We probably need to create some compatibility modules for the java.* modules being purged from the JDK.

Reply:
I can agree that making the change sooner is better however this adds additional amount of work required to be done - at least update all components covered by Eclipse Metro project (includes jaxb, jaxws, saaj and majority of its dependencies). Not saying that my understanding of https://github.com/eclipse-ee4j/ee4j/issues/34#issuecomment-436605211  is if it is there, do not touch it, if it is not there, do not add it

Reply:
If it's too much work, just use the old module.  The few people using this stuff with modules can use the non-Eclipse version of JAF that uses the old module name.  This doesn't impact GlassFish integration because JAF isn't included in GlassFish, it comes from JDK 1.8.